src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14180,19 +14331,19 @@ static void Main()
                 			native int 'method'
                 		) runtime managed 
                 	{
-                	} // end of method '<>f__AnonymousDelegate0'::.ctor
+                	} // end of method '<>f__AnonymousDelegate0`2'::.ctor
                 	.method public hidebysig newslot virtual 
                 		instance void Invoke (
-                			int32 arg1,
-                			int32[] arg2
+                			!T1 arg1,
+                			!T2 arg2
```

Comment:
>!T2 arg2 [](http://example.com/codeflow?start=19&length=8)

I think we should consider making this parameter ```!T2[] arg2``` and substitute T2 with array element type at the call site

Reply:
I've considered that in my [other comment](https://github.com/dotnet/roslyn/pull/65582#discussion_r1034639654). Is it worth the added complexity? The difference shouldn't be visible except for IL.

Reply:
Makes sense, thanks. Changed.